src/lib/OpenEXR/ImfStandardAttributes.h

```diff
@@ -299,17 +762,103 @@ IMF_STD_ATTRIBUTE_DEF (framesPerSecond, FramesPerSecond, Rational)
+// framelines -- set of rectangles used for creative purposes, e.g. to
+// indicate safe areas, cropping for derivative image sequences, etc
+//
+// The way in which the format encodes these rectangles is not defined;
+// the encoding might be camera-vendor-specific, or it might be formally
+// or informally standardized (e.g. an ASC Frameline Decision List)
+//
+
+IMF_STD_ATTRIBUTE_DEF (framelines, Framelines, std::string)
+
+
+//
+// imageRotation -- rotation of the image in degrees, in the closed interval
+// [-45, +45], with the rotation around the center of the display window, a
+// positive value specifying a rotation from x-axis to y-axis
+//
+
+IMF_STD_ATTRIBUTE_DEF (imageRotation, ImageRotation, float)
+
+
+//
+// cameraPosition -- x, y, z position of the camera, in meters
+//
+// The attribute may record the center of the camera's sensor, or the
+// center of the entrance pupil of the lens. The coordinate system for
+// the camera's position is fixed in relation to the set, and is a
+// right-handed Cartesian coordinate system with the z-axis pointing
+// upwards and the y-axis pointing 90 degrees to the left of the
+// x-axis direction.
+//
+// Recording the position of the center of the entrance pupil provides the
+// most usable value in some circumstances, e.g. as the camera pans. This
+// position changes, however, with lens and focus. For cases when this
+// precision is not needed, recording the center of the sensor is provided
+// as an alternative.
+//
+// For higher precision the chosen coordinate system is usually NOT the GPS
+// system, but an on-set coordinate system. Any coordinate system can be 
+// chosen for recording the camera's position, as long as its axes are
+// orthogonal, and z points up. The coordinate system can be moving with
+// the set, for example, when on a ship. Its reference point (usually
+// assignged position 0,0,0) can be within or outside the set.
+
+IMF_STD_ATTRIBUTE_DEF (cameraPosition, CameraPosition, IMATH_NAMESPACE::V3f)
+
+
+//
+// cameraUpDirection -- the 'up' direction of the camera
+//
+// If present, must be accompanied by a cameraViewingDirection attribute.
+// The 'up' direction should be perpendicular to the direction specified in the
+// cameraViewingDirection attribute. The coordinate system in which the
+// 'up' direction is expressed is that defined for the cameraViewingDirection
+// attribute.
+
+IMF_STD_ATTRIBUTE_DEF (cameraUpDirection, CameraUpDirection, IMATH_NAMESPACE::V3f)
+
+
+//
+// cameraViewingDirection -- the viewing direction of the camera, i.e. the
+// direction of the optical axis of the lens, as a right-handed Cartesian
+// coordinate system with the z-axis pointing upwards and the y-axis
+// pointing 90 degrees to the left of the x-axis direction.
+// 
+// If present, must be accompanied by a cameraUpDirection attribute.
+//
+// If the cameraPosition attribute is present, the axes of the
+// cameraViewingDirection attribute are parallel to, and point in the same
```

Comment:
As noted earlier, this coordinate system is not compatible with the routines existing in the Imath library. I think we should put a note here that says "swap these values, and negate that one to use this attribute with Imath functions."